config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-periodics-main.yaml

```diff
@@ -0,0 +1,34 @@
+periodics:
+- name: periodic-cluster-api-provider-vsphere-coverage
+  interval: 24h
+  decorate: true
+  path_alias: "sigs.k8s.io/cluster-api-provider-vsphere"
+  extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-vsphere
+      base_ref: main
+      path_alias: "sigs.k8s.io/cluster-api-provider-vsphere"
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+  spec:
+    containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220124-681f3531ec-master
+        command:
+          - runner.sh
+          - bash
+        args:
+          - -c
+          - |
+            result=0
+            ./scripts/ci-test-coverage.sh || result=$?
```

Comment:
I think the path of the file is incorrect, it should match the file path from the CAPV repo that has been included in ur PR

Reply:
@srm09 I have added the `ci-test-coverage.sh `to a new folder named scripts. Assuming that the reference point is CAPV directory, this path should work.
https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/1356/files#diff-88a9a164615a5c05ea7ac742264bc8fd8145978c5a30d3fc52f03b71b7e59c9bR1